dialog/file.go

```diff
@@ -365,6 +435,23 @@ func (f *fileDialog) setSelected(file *fileDialogItem) {
 	}
 }
 
+func (f *fileDialog) setView(view viewLayout) {
+	f.view = view
+	if f.view == gridView {
+		padding := fyne.NewSize(fileIconCellWidth-fileIconSize, theme.Padding())
+		f.files = fyne.NewContainerWithLayout(layout.NewGridWrapLayout(
```

Comment:
This should use `container.NewGridWrap()`, the function used above is deprecated.